include/picongpu/particles/ionization/byField/IonizationCurrent/IonizationCurrent.def

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+namespace picongpu
+{
+namespace ionizationCurrent
+{
+    struct EnergyConservation;
```

Comment:
Add doxygen documentation for all structs in the file. See e.g. documentation of BSI_impl.